.ci/pytorch/common_utils.sh

```diff
@@ -208,6 +208,28 @@ function clone_pytorch_xla() {
   fi
 }
 
+# install trition-lang/triton at a specific commit taken as argument
+function checkout_install_triton_at_commit() {
+
+  # assert that the commit is passed as an argument
+  if [ $# -ne 1 ]; then
+    echo "Usage: checkout_install_triton_at_commit <commit>"
+    exit 1
+  fi
+  local commit
+  commit=$1
+  pushd ..
+  git clone --recurse-submodules https://github.com/triton-lang/triton.git
+  pushd triton
+
+  git checkout "${commit}"
+  # taken from triton README
+  pip install ninja cmake wheel; # build-time dependencies
+  pip install -e python
+  popd
+  popd
```

Comment:
What's wrong with a single line
pip install --force-reinstall "git+https://github.com/trition-lang/trition@${commit}#subdirectory=python"

Also, why not reuse /extend install_trition_wheel.sh?

Reply:
I didn't realize you could do that. I'll modify install_triton_wheel to do this.